ci/docs/layout.html

```diff
@@ -12,4 +12,36 @@
   ga('send', 'pageview');
 
 </script>
+
+{# check current version against supported-releases.json #}
+<script>
+$( document ).ready( function() {
+
+  // gets version number minus whitespace and bugfix version
+  var $version_number = $('.version').text().trim().split('.', 2).join('.')
+
+  var supported_versions = []
+  // get JSON from server and load supported_versions
+  $.getJSON("https://raw.githubusercontent.com/pulp/pulp_packaging/master/ci/config/releases/supported-releases.json" , function(data) {
```

Comment:
Since this is an asynchronous request supported_versions is not guaranteed to be populated when ``if ($.inArray($version_number, supported_versions) == -1)`` is being checked. 
Consider adding the check in a .done() callback function like this: http://api.jquery.com/jquery.getjson/#jqxhr-object